jujugui/static/gui/src/app/app.js

```diff
@@ -908,9 +908,9 @@ YUI.add('juju-gui', function(Y) {
           controllerAPI, bakery)}
       />);
 
-      let logoutUrl = '/logout';
+      let logoutUrl = 'logout';
       if (window.juju_config.baseUrl) {
-        logoutUrl = window.juju_config.baseUrl + logoutUrl;
+        logoutUrl = window.juju_config.baseURL + logoutUrl;
```

Comment:
Even if we sanitize the value on the WSGI app, I think we have to still be defensive here as the GUI can be run outside of the WSGI wrapper (e.g. in gijoe). Also, here, when baseURL is not provided, the logout url is "logout", without the initial slash. What do you think about including my change at https://github.com/juju/juju-gui/pull/3001 in this branch?